lib/index.d.ts

```diff
@@ -2472,7 +2482,7 @@ interface Parser {
      * @param recognizer - A recognizer instance to determine node boundaries.
      * @returns A list of parsed `CssNode` objects.
```

Comment:
These tags have no effect now that the method syntax is not used, and since the same information is in the `ReadSequenceFunction` TSDoc we could probably remove them.